editor/index.js

```diff
@@ -45,17 +46,27 @@ window.jQuery( document ).on( 'heartbeat-tick', ( event, response ) => {
 /**
  * Initializes and returns an instance of Editor.
  *
+ * The return value of this function is not necessary if we change where we
+ * call createEditorInstance(). This is due to metaBox timing.
+ *
  * @param {String}  id       Unique identifier for editor instance
  * @param {Object}  post     API entity for post to edit
  * @param {?Object} settings Editor settings object
+ * @return {Object} Editor interface. Currently supports metabox initialization.
  */
 export function createEditorInstance( id, post, settings ) {
 	const target = document.getElementById( id );
 
-	render(
-		<EditorProvider settings={ settings } post={ post }>
+	const provider = render(
+		<EditorProvider settings={ settings } post={ post } target={ target }>
```

Comment:
I guess padding the `target` is useless here? (probably a rebase thing)

Reply:
Yes, I wasn't sure what happened in the branch, I will have to go look at it.

Reply:
Fixed it, sorry about that 🙃